iCubGenova04/calibrators/left_arm-calib.xml

```diff
@@ -23,7 +23,7 @@
 <param name="calibration4">           0          0          0          0            0     0      0  2170   200   241   499   255   465    226    480  694 </param>
 <param name="calibration5">           0          0          0          0            0     0      0  2580  2372   46    22     16   10     33      0   135 </param>
 <param name="calibrationZero">        -180.00    -315.00    180.00     -180.00      0  -180    180     0     0     0     0     0     0      0      0     0 </param>
-<param name="calibrationDelta">       -1.4       -10.1      -15.7      -4.3         0     0      0     0     0     0     0     0     0      0      0     0 </param>
+<param name="calibrationDelta">        -66.5      -10.4     -16.2        -1.8       0     0      0     0     0     0     0     0     0      0      0     0 </param>
```

Comment:
Wow.. This is quite a big offset!

Reply:
Yes indeed. But it was already in the local configuration of the robot before my changes. The actual offset I added was of around **1deg**. I didn't have time to dig into the history of this change, but I imagine it was the result of some fix in the shoulder. The last update on the source files of this joint encoder offset was done by @S-Dafarra on the 15 June, commit fe7e538.
Ciao @julijenv , did you have to change the original position of one or several encoders on the left shoulder on the green iCub?

Reply:
Maybe it happened after https://github.com/robotology/icub-support/issues/506.